src/ray/raylet/reconstruction_policy.cc

```diff
@@ -130,39 +162,114 @@ void ReconstructionPolicy::AttemptReconstruction(const TaskID &task_id,
   it->second.reconstruction_attempt++;
 }
 
+bool ReconstructionPolicy::CheckExpiredTask(const Task &task,
+                                            int64_t lease_actor_version) {
+  if (task.GetTaskSpecification().IsActorTask()) {
+    // The task is an actor task. Check its version against the executed
+    // version, according to the task lease table, and the current version of
+    // the actor, according to the actor table. This will determine whether:
+    //   (1) the actor is still alive and may execute the task.
+    //   (2) the actor executed the task but its result has been evicted.
```

Comment:
What happens if the actor task had multiple outputs and only some of them have been evicted?